src/HID-APIs/SurfaceDialAPI.h

```diff
@@ -49,17 +52,30 @@ class SurfaceDialAPI
   inline void end(void);
   inline void click(void);
   inline void rotate(int16_t rotation);
+  inline void position(int16_t x, int16_t y);
```

Comment:
W.r.t. the data type - shouldn't the X and Y coordinates be unsigned?

Reply:
but isnt 0, 0 the screen center?